databricks_cli/runs/cli.py

```diff
@@ -65,23 +67,30 @@ def _runs_to_table(runs_json):
               help='If specified, only active runs will be listed')
 @click.option('--completed-only', is_flag=True, default=None,
               help='If specifed, only completed runs will be listed')
-@click.option('--offset', default=None, type=int)
-@click.option('--limit', default=None, type=int)
-@click.option('--output', default=None, help=OutputClickType.help, type=OutputClickType())
+@click.option('--offset', default=None, type=int,
+              help='Set to 0 by default. The offset is relative to the most recent run ID.')
+@click.option('--limit', default=None, type=int,
+              help='Set to 20 runs by default. The limit determines the number of runs listed. '
+                   'Limit must be between 0 and 1000.')
+@click.option('--output', help=OutputClickType.help, type=OutputClickType())
 @require_config
-@eat_exceptions
+@eat_exceptions # noqa
```

Comment:
What does `noqa` mean?

Reply:
Tells the linter to avoid this line. For some reason, I was getting a "too many parameters" lint on it otherwise.